### DISCUSSION TITLE
Thumbnail with webp

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [pimcore/pimcore#14734] [Q&A / Support]
Hi,

I have checked on demo website there we have selected webp in the product_detail thumbnail but still images are loading as jpg only.

**Comment by NiklasBr** [comment]
Is your browser sending the `Accept: image/webp` request header?
![image](https://user-images.githubusercontent.com/279826/227263549-6a468c08-1c2c-4442-8494-3f3ab8146f53.png)
